### PR TITLE
Added five new rows to r_instat_menus JSON file

### PR DESCRIPTION
**Pull request by N-thony**
@lloyddewit this is an updated en_r_instat_menus file with new entries

### CODE REVIEW

**Comment by lloyddewit** [comment]
@N-thony Thank you for the changes. The file we want to update is `translations/en/r_instat_menus.json`. This PR is trying to update the file in an older location. If we merge this PR then it will just add a new JSON file in the wrong location.
Please could you correct? thanks

**Comment by N-thony** [comment]
@lloyddewit I think have resolved your comment

File: instat/translations/en/r_instat_menus.json [hunk 1]
```diff
@@ -399,5 +399,11 @@
     "More...":"More...",
     "About R-Instat":"About R-Instat",
     "Licence...":"Licence...",
+    "Acknowlegments":"Acknowlegments",
```
Comment (lloyddewit) [inline instat/translations/en/r_instat_menus.json:402]: This line is included twice.
I also noticed that it is spelled wrong, it should be 'Acknowledgments'. Please could you correct it in the file? I will correct it in the R-Instat code, thanks

File: instat/translations/en_r_instat_menus.json [hunk 2]
```diff
@@ -0,0 +1,408 @@
+{
```
Comment (lloyddewit) [inline instat/translations/en_r_instat_menus.json:1]: This PR should only contain the file above. Please could you remove this file from the PR? thanks

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@maxwellfundi @shadrackkibet please could one of you approve and merge? thanks

**Review by dannyparsons — APPROVED** [review]

**Pull request merged**